doc/release/RELEASE-PROTOCOL.md

```diff
@@ -134,27 +134,44 @@ Make sure that:
+9. Upon completing the release, the responsible person must notify stakeholders in the *#data_x_product* Slack channel that the releases to sandbox and production have concluded. Additionally, update the *Unreleased* version's status in Jira.
+10. As the final step, shut down the pre-prod environment, as it is no longer required for the current version of the release.
 
 ### Hotfixes
 
-- To deploy a hotfix to pre-prod, you should fork from the latest release branch, and after preparing the fix, merge it back. Merging will trigger the Deploy to AWS workflow that will deploy the hotfix to the **running** pre-prod environment. Before doing it, don't forget about the step involving switching databases, as mentioned in [the Code Freeze section]((#requirements-and-key-results)).
-- To release a hotfix to production and staging, you should fork from the latest release branch, and after preparing the fix, merge it back. The last step is to execute the Release [Deploy] workflow for each environment separately, which will deploy the fix to these two environments.
+- To deploy a hotfix to pre-prod, you should fork from the latest release branch, and after preparing the fix, merge it back. Merging will trigger the Deploy to AWS workflow that will deploy the hotfix to the **running** pre-prod environment.
+- To release a hotfix to production and staging, you should fork from the latest release branch, and after preparing the fix, merge it back. The last step is to execute the `Release [Deploy]` workflow for each environment separately, which will deploy the fix to these two environments.
 
 ### Shut down the pre-prod environment
 
-- To shut down the Pre-prod the responsible person have to select `Destroy Environment` in the Actions menu. Press run workflow with `main` branch and select `Pre-prod` env.
-- DB Instance has to be destroyed manually via RDS -> Databases -> `opensupplyhub-enc-pp`. Press Modify -> Enable deletion protection=false. Then select Delete in Actions.
+1. Shutting down the pre-prod environment starts after passing all QA checklist items for the pre-prod.
+2. First step is to destroy pre-prod DB Instance manually via RDS -> Databases -> `opensupplyhub-enc-pp`. Press Modify -> Enable deletion protection=false. Then select Delete in Actions.
+3. Finally, to shut down the Pre-prod the responsible person have to select `Destroy Environment` in the Actions menu. Press run workflow with `main` branch and select `Pre-prod` env.
+
+### Post Release Notes:
+
+On Monday after each release, current metrics should be checked by QA engineer.
+
+1. Kamino:
+
+    - Successfully logged with valid OS HUb admin credentials and re-directed to Kamino's [main page](https://34.241.25.221/kamino/bk).
+2. Looker:
+
+    - `duplicate_ratio_perc` ~ 2 (+- 0.1)
+
+    - `estimated_duplicates` ~ 4000-6000
+3. Airflow:
+
+    - Dag_Id duplillom should approximately take 50 min. So we can conclude that it <strong>must be > 2.5</strong>
 
 ### Reloading the DedupeHub
```

Comment:
Please add it to the menu at the top of the file.

Reply:
**Reloading the DedupeHub** item is already added to the top

Reply:
It should be  `- [Reloading the DedupeHub](#reloading-the-dedupehub)` but not  `- [Reloading the DedupeHub](#reload-dedupehub)`
After fixing the link, it will forward a user directly to the "Reloading the DedupeHub" section upon clicking
<img width="1290" alt="Screenshot 2024-05-21 at 16 38 41" src="https://github.com/opensupplyhub/open-supply-hub/assets/58369841/4eb3db76-b523-4ac9-838b-b50dc32d61a2">

Reply:
Yea, I've noticed this now, thank you. Fixed.